launch/threshold.launch

```diff
@@ -0,0 +1,62 @@
+<launch>
+  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
+
+  <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
+  <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
```

Comment:
please support argument like 
https://github.com/ros-perception/opencv_apps/blob/indigo/launch/edge_detection.launch#L9-L17

and put 1 node at launch file and for different threshold type example, put them within .test code